project.ncl

```diff
@@ -7,7 +7,6 @@ let import_nix = organist.nix.import_nix in
     organist.shells.Bash
     & {
       build.packages = {
-        nickel-lang-cli = import_nix "nixpkgs#nickel",
```

Comment:
I don't know if it's related, but the CI is failing (with a not very informative error). How do build packages and flake packages interact? Is it ok to remove this one?

Reply:
Mh, maybe not.

But that's also not the reason for the CI failure. The CI just complains that `nls` doesn't accpet a `--version` flag :D

```
+ nls --version
error: unexpected argument '--version' found
```

Reply:
Argh, this is embarrassing.